From 0532ce497399a8f694da8d6e4b289685f40698b2 Mon Sep 17 00:00:00 2001
From: Teknoman117 <linux.robotdude@gmail.com>
Date: Wed, 24 Jul 2013 03:00:18 -0700
Subject: [PATCH 2/4] Add the clock bindings to omap3.dtsi that were made
 available through the omap-clock device tree driver.
 Adds the pll1 clock handles which drives the CPU
 frequency, and subsequently the ability to change its
 clock speed, and also add the system reference clock.
 Originally found here:
 http://lkml.indiana.edu/hypermail/linux/kernel/1304.1/04074.html

---
 arch/arm/boot/dts/omap3.dtsi |   12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm/boot/dts/omap3.dtsi b/arch/arm/boot/dts/omap3.dtsi
index 7d95cda..562f51e 100644
--- a/arch/arm/boot/dts/omap3.dtsi
+++ b/arch/arm/boot/dts/omap3.dtsi
@@ -30,6 +30,8 @@
 
 		cpu@0 {
 			compatible = "arm,cortex-a8";
+			clocks = <&dpll1>;
+			clock-names = "cpu";
 			device_type = "cpu";
 			reg = <0x0>;
 		};
@@ -41,6 +43,16 @@
 		ti,hwmods = "debugss";
 	};
 
+	sysclk: sys {
+		#clock-cells = <0>;
+		compatible = "ti,omap-clock";
+	};
+
+	dpll1: dpll1 {
+		#clock-cells = <0>;
+		compatible = "ti,omap-clock";
+	};
+
 	/*
 	 * The soc node represents the soc top level view. It is used for IPs
 	 * that are not memory mapped in the MPU view or for the MPU itself.
-- 
1.7.10.4

